### PR TITLE
feat: add weETHs logo and metadata

### PR DESCRIPTION
**Pull request by tkporter**
### Description

prep for the deployment of a warp route of https://etherscan.io/address/0x917cee801a67f933f2e6b33fc0cd1ed2d5909d88#code to Eclipse

### CODE REVIEW

File: deployments/warp_routes/weETHs/logo.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<circle cx="29.9949" cy="29.9949" r="27.675" stroke="#120370" stroke-width="1.05"/>
+<circle cx="29.9949" cy="29.9949" r="27.675" stroke="url(#paint0_linear_2142_61194)" stroke-width="1.05"/>
+<defs>
+<linearGradient id="paint0_linear_2142_61194" x1="29.5712" y1="57.2114" x2="41.5282" y2="9.12662" gradientUnits="userSpaceOnUse">
+<stop stop-color="#29BCFA"/>
+<stop offset="0.43" stop-color="#6464E4"/>
+<stop offset="1" stop-color="#B45AFA"/>
+</linearGradient>
+</defs>
+</svg>
```
Comment (paulbalaji) [inline deployments/warp_routes/weETHs/logo.svg:13]: minified even more https://jakearchibald.github.io/svgomg/
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 60 60"><rect width="60" height="60" fill="#0E0B31" rx="30"/><path fill="#B6FF3E" fill-rule="evenodd" d="M44.4 43.9c0 .3 0 .4-.3.5L30.2 53h-.6l-13.8-8.4a.6.6 0 0 1-.2-.6V33.3a.6.6 0 0 1 .8-.6L30 40.7l13.4-7.8.4-.2c.4 0 .6.3.6.6v10.5ZM16.7 43l4-2.4-4-5.5v8Zm4.7-1.4L17.3 44 28 50.7l-6.7-9.1Zm10.6 9L42.7 44l-4.1-2.5-6.6 9Zm7.3-10 4 2.3v-8l-4 5.6Zm-9.6 1.2-11.5-6.6L30 51.4l11.6-16.1-11.3 6.5h-.6ZM16.1 30.5a.6.6 0 0 1-.5-.5V16c0-.2 0-.4.2-.5l14-8.5h.6L44 15.4c.2.1.3.3.3.5v14c0 .2-.1.4-.4.5L30.3 38h-.6l-13.6-7.5Zm.6-13.5v11.1l5-8-5-3Zm26.6 0-5 3 5 8.1V17Zm-5.7 2 5.1-3.1-11.1-6.8 6 9.9Zm-9.3-9.8-11 6.8 5 3 6-9.8ZM17.1 29.7 30 36.8l12.9-7-13-21-12.8 21Z" clip-rule="evenodd"/><circle cx="30" cy="30" r="27.7" stroke="#120370" stroke-width="1.1"/><circle cx="30" cy="30" r="27.7" stroke="url(#a)" stroke-width="1.1"/><defs><linearGradient id="a" x1="29.6" x2="41.5" y1="57.2" y2="9.1" gradientUnits="userSpaceOnUse"><stop stop-color="#29BCFA"/><stop offset=".4" stop-color="#6464E4"/><stop offset="1" stop-color="#B45AFA"/></linearGradient></defs></svg>
```

File: deployments/warp_routes/weETHs/metadata.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Super Symbiotic LRT",
```
Comment (paulbalaji) [inline deployments/warp_routes/weETHs/metadata.json:2]: is this the right name for weETH

Comment (daniel-savu) [reply deployments/warp_routes/weETHs/metadata.json:2]: yes https://discord.com/channels/935678348330434570/1293951457577664582/1303374426074517625

**Review by daniel-savu — APPROVED** [review]

File: deployments/warp_routes/weETHs/logo.svg (hunk 1 above)
Comment (tkporter) [reply deployments/warp_routes/weETHs/logo.svg]: nice

File: deployments/warp_routes/weETHs/metadata.json (hunk 2 above)
Comment (tkporter) [reply deployments/warp_routes/weETHs/metadata.json:2]: https://etherscan.io/address/0x917cee801a67f933f2e6b33fc0cd1ed2d5909d88#readContract#F7

**Pull request merged**